.travis.yml

```diff
@@ -99,68 +99,108 @@ geosx_pangea_build: &geosx_pangea_build
   - gcloud auth activate-service-account --key-file=${GEOSX_GCLOUD_KEY}
   - gsutil cp -a public-read ${GEOSX_BUNDLE} gs://geosx/Pangea2/
 
+draft_script: &draft_script
+  script:
+  # TRAVIS_PULL_REQUEST is false if job is not from a PR
+  - if [[ $TRAVIS_PULL_REQUEST == false ]]; then exit 0; fi;
+  - |
+    is_draft=$(curl -H "Accept: application/vnd.github.v3+json" \
+        https://api.github.com/repos/GEOSX/GEOSX/pulls/$TRAVIS_PULL_REQUEST | \
+        jq ".draft")
+
+  # CI jobs will be cancelled if PR is a draft.
+  # PR status must be "Open" to run CI.
+  - |
+    if [[ $is_draft == true ]]; then
+      exit 1
+    else
+      exit 0
+    fi
+
+return_script: &return_script
+  script:
+  - |
+    if [[ $(curl -sS -H "Travis-API-Version: 3" -X \
+          GET https://api.travis-ci.com/build/$TRAVIS_BUILD_ID/jobs | \
```

Comment:
The option `-X` has for argument `GET`. It would be better not to split in between.

Also, `curl` uses `GET` by default so writing `-X GET` is a (bad/wrong?) habit I took to be sure not to forget to change `GET` into `POST`, `PUT`, `PATCH` or `DELETE` when needed. But we could remove it there.

Reply:
I like the explicit form rather than relying on the default.  I think your habit to include it is probably a good habit.